modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/FieldBase/ReactFieldBase.es.js

```diff
@@ -146,4 +146,37 @@ describe('ReactFieldBase', () => {
 
 		expect(container).toMatchSnapshot();
 	});
+
+	it('renders the the add when repeatable is true', () => {
```

Comment:
@alinedoleron, the word `the` is repeated. Maybe we should say `renders the add button when repeatable is true`.